src/app/Sankalan/Quiz/response.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+use \DUCS\Sankalan\Database;
+
+/**
+ * returns problems json based on event
+ * @param  string $event event id
+ * @return array        list of problems
+ */
+function saveResponse($content)
+{
+    $params =  json_decode($content, true);
+    if (isset($params['ques_id']) and isset($params['event_id']) and isset($params['team_id']) and isset($params['attempt'])) {
+        $db = new Database();
+        $response = $db->modify('INSERT INTO submissions (tid, eid, qid, attempt)

```

Comment:
lets follow this approach:
we will only `UPDATE` here. All `INSERT` are done at the time when the questions are assigned